package.json

```diff
@@ -30,23 +30,29 @@
   ],
   "private": true,
   "scripts": {
-    "build": "yarn run babel && yarn run type:dts",
-    "build-one": "nimbus babel --clean",
     "babel": "yarn run babel:cjs && yarn run babel:esm",
     "babel:cjs": "nimbus babel --clean --workspaces=\"@visx/!(demo)\"",
     "babel:esm": "nimbus babel --clean --workspaces=\"@visx/!(demo)\" --esm",
+    "build": "yarn run babel && yarn run type",
+    "build-one": "echo 'build-one has been replaced with build:workspace' & exit 1",
+    "build:sizes": "yarn run ts ./scripts/computeBuildSizes.ts",
+    "build:workspace": "nimbus babel --clean",
+    "check:sizes": "yarn run ts ./scripts/compareBuildSizes.ts",
     "clean": "rm -rf ./packages/**/{lib,esm}",
     "format": "yarn run prettier --write",
     "jest": "NODE_ENV=test nimbus jest --coverage --verbose",
     "lint": "nimbus eslint",
     "lint:fix": "yarn run lint --fix",
+    "prepare-release": "git checkout master && git pull --rebase origin master && lerna updated",
     "prettier": "nimbus prettier",
+    "release": "yarn run prepare-release && lerna publish --exact",
     "setup": "yarn run build",
     "test": "yarn run jest",
-    "type:dts": "nimbus typescript --build --reference-workspaces",
-    "type-one": "nimbus typescript --build",
-    "prepare-release": "git checkout master && git pull --rebase origin master && lerna updated",
-    "release": "yarn run prepare-release && lerna publish --exact"
+    "ts": "ts-node --project ./tsconfig.node.json",
```

Comment:
`ts-node` allows running TypeScript scripts. requires the TS config to understand `commonjs` module format.